app/lib/s3-client.ts

```diff
@@ -0,0 +1,30 @@
+import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
+import { v4 as uuidv4 } from 'uuid';
+
+const s3Client = new S3Client({
+  region: process.env.AWS_REGION || 'us-east-1',
+  credentials: {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID!,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY!,
+  },
+});
+
+export async function uploadPDFToS3(pdfBuffer: Uint8Array): Promise<string> {
+  const uuid = uuidv4();
+  const bucketName = process.env.S3_BUCKET_NAME || 'vargas-jr-memory';
+  const key = `contracts/${uuid}.pdf`;
```

Comment:
We should create a utility to generate a S3 key. In that utility:
- If the environment is production, no-op, and just return the key
- If the environment is preview, prefix with `previews/${previewId}/`

Reply:
In future PRs, all S3 keys should go through this utility